services-js/311/components/common/TelephoneNumbers.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { css } from 'emotion';
+
+import { assetUrl, MEDIA_LARGE } from '../style-constants';
+
+const SPRITE_URL = assetUrl('img/svg/faq-icons.svg');
+
+const CONTAINER_STYLE = css({
+  display: 'flex',
+  alignItems: 'center',
+});
+
+const TELEPHONE_STYLE = css({
+  display: 'none',
+  width: 36,
+  height: 36,
+  marginRight: '1rem',
+  [MEDIA_LARGE]: {
+    display: 'block',
+  },
+});
+
+/**
+ * Component that adds a little telephone icon and information about how to call
+ * BOS:311 when the website is having issues.
+ */
+export default function TelephoneNumbers(): JSX.Element {
+  return (
+    <div className={CONTAINER_STYLE}>
+      <svg role="img" className={TELEPHONE_STYLE}>
+        <use xlinkHref={`${SPRITE_URL}#Phone_off`} height="36" />
+      </svg>
+
+      <div className="t--info">
+        You can always report non-emergency issues to us by calling BOS:311 at
+        311 or <span style={{ whiteSpace: 'nowrap' }}>617-635-4500</span>.
```

Comment:
```suggestion
        311 or <a href="tel:+16176354500" style={{ whiteSpace: 'nowrap' }}>617-635-4500</a>.
```
This would make life easier for mobile users, and improve accessibility.

Reply:
Done. Also updated the translate dialog to include a similar link.